ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -234,12 +240,14 @@ export const WorkspaceCard = fp.flow(withNavigation)(
             >
               <FlexColumn style={{marginBottom: 'auto'}}>
                 <FlexRow style={{ alignItems: 'flex-start' }}>
-                  <Clickable>
+                  <Clickable style={{cursor: disabled ? 'not-allowed' : 'pointer', ...styles}} disabled={disabled}>
+                    <TooltipTrigger content={'Controlled Tier workspace is not accessible.'} disabled={!disabled}>
                     <div style={styles.workspaceName}
                          data-test-id='workspace-card-name'
-                         onClick={() => this.onClick()}>
+                         onClick={() => !disabled ? this.onClick() : {}}>
```

Comment:
It shouldn't be necessary to check `disabled` explicitly.  Can we do something like moving the `onClick` to the `Clickable`?